src/py42/modules/detectionlists.py

```diff
@@ -5,3 +5,37 @@ def __init__(self, microservice_client_factory):
     @property
     def departing_employee(self):
         return self._microservice_client_factory.get_departing_employee_client()
+
+    @property
+    def high_risk_employee(self):
```

Comment:
all the user methods should get the detection_list_user_client by calling `self._microservice_client_factory.get_detection_list_user_client()` once it exists.

Reply:
Done.